front-end-admin/src/components/callout/CalloutList.vue

```diff
@@ -2,7 +2,7 @@
   <div class="jumbotron">
     <div class="container">
       <b-card>
-      <router-link class="btn btn-default" to="/callout/register">Add Callout&raquo;</router-link>
+      <router-link class="btn btn-default" style="background-color: grey; color:white" to="/callout/register">Add Callout&raquo;</router-link>
```

Comment:
Instead of using the in-line style tag make this its own CSS class. Seeing as how this style is re-used across several components, I would make this a global class that is re-usable